UnityProject/Assets/Scripts/Player/PlayerSync.cs

```diff
@@ -496,10 +496,8 @@ private void Synchronize()
 			{
 				CheckMovementServer();
 			}
-			else
-			{
-				CheckMovementClient();
-			}
+
+			CheckMovementClient(); //needed for space floating movement to update for the Server if it is not headless
```

Comment:
```suggestion

                        else if (CustomNetworkManager.IsHeadless == false)
			{
			    CheckMovementClient(); //needed for space floating movement to update for the Server if it is not headless
			}
```

Think thats the headless check, or is this needed for headless too bit confused by what you mean by movement

Reply:
shall I run some headless tests?

Reply:
yeah, would be for the best

Reply:
done